### PR TITLE
ImageNet example: lmdb filename not consistent

### PR DESCRIPTION
**Pull request by drohit92**
The create_imagenet_mean.sh file creates 2 db files: ilsvrc12_train_lmdb and ilsvrc12_val_lmdb
But the make_imagenet_mean.sh is currently using ilsvrc12_train_leveldb. The names are not consistent. 
Have changed ilsvrc12_train_leveldb to ilsvrc12_train_lmdb
The names in the documentation example have to be also changed accordingly.

Doing a change for the first time. Plz let me know if I miss any protocol.
Thanks

### CODE REVIEW

File: examples/imagenet/make_imagenet_mean.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Compute the mean image from the imagenet training leveldb
```
Comment (seanbell) [inline examples/imagenet/make_imagenet_mean.sh]: Perhaps this comment should be updated to `lmdb`

Comment (drohit92) [inline examples/imagenet/make_imagenet_mean.sh]: @seanbell yes! did that

File: examples/imagenet/make_imagenet_mean.sh [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env sh
-# Compute the mean image from the imagenet training leveldb
+# Compute the mean image from the imagenet training lmdb
```
Comment (drohit92) [inline examples/imagenet/make_imagenet_mean.sh]: @seanbell  changed it.. thanks

**Comment by shelhamer** [comment]
Thanks for the fix -- I merged the same fix in #2055 because I happened to see it there first. Your PR was just fine though, so please do send fixes as you find them in the future!

**Pull request closed without merging**